README.md

```diff
@@ -1,7 +1,5 @@
 # TinyMCE documentation
 
-[![wercker status](https://app.wercker.com/status/4d4c743635332430f9d25acae1be5218/s/main "wercker status")](https://app.wercker.com/project/bykey/4d4c743635332430f9d25acae1be5218)
-
 This project maintains the documentation for TinyMCE at
 [https://www.tiny.cloud/docs](https://www.tiny.cloud/docs). If you have any
```

Comment:
@MitchC1999 should we update this to `https://www.tiny.cloud/docs/tinymce/latest/`?

Reply:
No. That URL will (hopefully) eventually point to a docs homepage, for all versions and products, so it is still an appropriate link. I feel like I mentioned this at some point.

Reply:
Now you mention it, i believe we did chat about this #noted4later.